SUPPORT.md

```diff
@@ -1,7 +1,6 @@
 # Harvest Support and Getting Help
 
-Welcome to Harvest! Harvest is an official NetApp open-source project. You can reach out
-to NetApp using any of the [standard mechanisms](http://mysupport.netapp.com/info/web/ECMLP2619434.html) and get support.
+Harvest is an open-source project developed and published by NetApp to collect performance, capacity and hardware metrics from ONTAP clusters. These metrics can then be delivered to a range of databases and displayed in Grafana dashboards. Harvest is not a NetApp product and as such is not supported the same way by NetApp. NetApp does maintain and update Harvest, which includes bug fixes and updates. For assistance refer [Getting Help](#getting%20help)
```

Comment:
instead of 
> Harvest is not a NetApp product and as such is not supported the same way by NetApp. NetApp does maintain and update Harvest, which includes bug fixes and updates.

what about

Harvest is not an officially supported NetApp product. NetApp maintains and updates Harvest with bug fixes, security updates, and feature development.

Reply:
done.